### PR TITLE
Fix 0.29 regressions

### PR DESCRIPTION
**Pull request by arifogel**

### CODE REVIEW

**Comment by dhalperi** [comment]
I pushed a recompiled version of the jars that still passes, just so we can be absolutely sure it's compiled from the ref in the BUILDING.md file.

File: projects/question/src/main/java/org/batfish/question/jsonpath/NodesPathQuestionPlugin.java [hunk 1]
```diff
@@ -1,289 +0,0 @@
-package org.batfish.question.jsonpath;
```
Comment (dhalperi) [inline projects/question/src/main/java/org/batfish/question/jsonpath/NodesPathQuestionPlugin.java]: Did you delete this file intentionally?

Comment (arifogel) [inline projects/question/src/main/java/org/batfish/question/jsonpath/NodesPathQuestionPlugin.java]: Yes. We don't use it.

Comment (arifogel) [inline projects/question/src/main/java/org/batfish/question/jsonpath/NodesPathQuestionPlugin.java]: It is a projected version of JsonPathQuestionPlugin. It only existed for historical reasons, as JsonPathQuestionPlugin is more general. All references to it are gone.

Comment (arifogel) [inline projects/question/src/main/java/org/batfish/question/jsonpath/NodesPathQuestionPlugin.java]: Well I was wrong. Back it goes.

**Pull request merged**